netmiko/nokia/nokia_sros.py

```diff
@@ -244,31 +259,51 @@ class NokiaSrosTelnet(NokiaSros):
 
 class NokiaSrosFileTransfer(BaseFileTransfer):
     def __init__(
-        self, ssh_conn, source_file, dest_file, hash_supported=False, **kwargs
-    ):
+        self,
+        ssh_conn: BaseConnection,
+        source_file: str,
+        dest_file: str,
+        file_system: Optional[str] = None,
+        direction: str = "put",
+        socket_timeout: float = 10.0,
+        progress: Optional[Callable[..., Any]] = None,
+        progress4: Optional[Callable[..., Any]] = None,
+        hash_supported: bool = True,
+    ) -> None:
         super().__init__(
-            ssh_conn, source_file, dest_file, hash_supported=hash_supported, **kwargs
+            ssh_conn,
+            source_file,
+            dest_file,
+            file_system,
+            direction,
+            socket_timeout,
+            progress,
+            progress4,
+            hash_supported,
```

Comment:
Needs to be named arguments as I wouldn't want to count on the order of arguments here (for anything after the first three arguments).